autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py

```diff
@@ -148,6 +148,8 @@ def get_loader(self, X: np.ndarray, y: Optional[np.ndarray] = None, batch_size:
         dataset = BaseDataset(
             train_tensors=(X, y),
             # This dataset is used for loading test data in a batched format
+            seed=self.random_state.get_state()[1][0],
+            shuffle=False,
             train_transforms=self.test_transform,
             val_transforms=self.test_transform,
         )
```

Comment:
```suggestion
        # The seed set for this `np.random.RandomState`
        seed = self.random_state.get_state()[1][0]
        dataset = BaseDataset(
            train_tensors=(X, y),
            # This dataset is used for loading test data in a batched format
            seed=seed,
            shuffle=False,
            train_transforms=self.test_transform,
            val_transforms=self.test_transform,
        )
```

Reply:
I do not agree with creating an extra variable. It is unnecessary and it does not make it clearer unless I am missing another pro in the code change.